config/default/kustomization.yaml

```diff
@@ -39,9 +39,39 @@ patchesStrategicMerge:
 #- manager_prometheus_metrics_patch.yaml
 
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
-#- manager_webhook_patch.yaml
+- manager_webhook_patch.yaml
 
 # [CAINJECTION] Uncomment next line to enable the CA injection in the admission webhooks.
 # Uncomment 'CAINJECTION' in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
 # 'CERTMANAGER' needs to be enabled to use ca injection
-#- webhookcainjection_patch.yaml
+- webhookcainjection_patch.yaml
+
+# the following config is for teaching kustomize how to do var substitution
```

Comment:
Why did you move this?

Reply:
I was getting this error
```console
$ kustomize build config/default/
Error: var '{SERVICE_NAME ~G_v1_Service {metadata.name}}' cannot be mapped to a field in the set of known resources
```

besides, kubebuilder creates it inside this directory, so i moved it there

Reply:
Ah, thanks. Was this maybe generated by an earlier version of kubebuilder, perhaps? I'm definitely 👍 to match how/where kubebuilder generates things.